melosys-eessi-app/src/main/java/no/nav/melosys/eessi/identifisering/OppgaveEndretHendelse.java

```diff
@@ -40,6 +41,7 @@ public boolean harMetadataRinasaksnummer() {
     public static class Ident {
         private String identType;
         private String verdi;
+        @ToString.Exclude
```

Comment:
Ok, men må ikke klassen annoteres med `@ToSring` for at metoden skal autogeneres med lombok?

Reply:
Nei, Lombok sin `@Data` legger til mye rart for deg 😄  Er litt magi om man ikke kjenner til det.
https://projectlombok.org/features/Data
>A shortcut for @ToString, @EqualsAndHashCode, @Getter on all fields, @Setter on all non-final fields, and @RequiredArgsConstructor!